lib/ReactViews/Custom/parseCustomHtmlToReact.js

```diff
@@ -26,6 +29,24 @@ const shouldProcessEveryNodeExceptWhiteSpace = function(node) {
 
 let keyIndex = 0;
 
+const ExternalLinkIcon = styled(StyledIcon).attrs({
+  glyph: Icon.GLYPHS.externalLink,
+  styledWidth: "10px",
+  styledHeight: "10px",
+  displayInline: true
+})`
+  margin-left: 5px;
+  fill: ${props => props.theme.colorPrimary};
+`;
+
+const externalIcon = React.createElement(ExternalLinkIcon, {});
```

Comment:
I think you should create this element inside the `processNode` call each time we need it - re-using the same react element might result in unexpected behavior.